### ISSUE TITLE
Jboss 7.1 final erro Reflection 0.9.6

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NickstaDB/reflections#105]
```
Hi all..
I need migrate my web aplication to jboss 7.1.
I used reflections 0.9.5 in tomcat 6. But.. the same code, using jboss 7.1 and 
reflections 0.9.5 ou 0.9.6.. not work.

with reflections 0.9.6 happenings this:

http://pastie.org/3479494

What´s wrong?

tanks all.. 

ps: this project very usefull.. tanks all
:)
```

Original issue reported on code.google.com by `coelh...@gmail.com` on 28 Feb 2012 at 12:26

**Comment by GoogleCodeExporter** [comment]
```
fixed on 0.9.7-RC1 and trunk. comments?
```

Original comment by `ronm...@gmail.com` on 2 May 2012 at 3:45
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed 0.9.7
```

Original comment by `ronm...@gmail.com` on 22 May 2012 at 7:47
- Changed state: **Fixed**
